content/docs/demos/prometheus_grafana.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Manual deployment of OSM's Prometheus and Grafana stack"
+title: "Prometheus & Grafana"
 description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
 type: docs
+weight: 1
```

Comment:
Let's set this to the highest among the current demos, given the integration with Prometheus and Grafana is optional.